src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+---
+title: 'OpenTelemetry tutorials: Instrument a sample Java app'
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: You can set up OpenTelemetry to monitor a simple Java app.
+---
+
+import opentelemetrySpringBootASCIIDisplay from 'images/opentelemetry_screenshot-crop_Spring-Boot-ASCII-display.png'
+
+import opentelemetryMainDistributedTracingPage from 'images/opentelemetry_screenshot-crop_main-distributed-tracing-page.png'
+
+import opentelemetryDistributedTracingError from 'images/opentelemetry_screenshot-crop_distributed-tracing-error.png'
+
+import opentelemetryInProcessSpans from 'images/opentelemetry_screenshot-crop_in-process-spans.png'
+
+import opentelemetryExceptiononSpanEvent from 'images/opentelemetry_screenshot-crop_exception-on-span-event.png'
+
+import opentelemetryAttributesTab from 'images/opentelemetry_screenshot-crop_attributes-tab.png'
+
+import opentelemetryMetricsExplorerMain from 'images/opentelemetry_screenshot-crop_metrics-explorer-main.png'
+
+import opentelemetryMetricsExplorerDimensions from 'images/opentelemetry_screenshot-crop_metrics-explorer-dimensions.png'
+
+import opentelemetryJvmsPageMain from 'images/opentelemetry_screenshot-crop_jvms-page-main.png'
+
+import opentelemetryCompareJvmInstances from 'images/opentelemetry_screenshot-full_compare-jvm-instances.png'
+
+import opentelemetryJvmColorCodedComparison from 'images/opentelemetry_screenshot-full_jvm-color-coded-comparison.png'
+
+import opentelemetryMainLogsPage from 'images/opentelemetry_screenshot-full_main-logs-page.png'
+
+import opentelemetryLogsinTerminal from 'images/opentelemetry_screenshot-full_logs-in-terminal.png'
+
+import opentelemetryLogDetailsWithTrace from 'images/opentelemetry_screenshot-full_log-details-with-trace.png'
+
+import opentelemetryTraceRelatedtoLog from 'images/opentelemetry_screenshot-full_trace-related-to-log.png'
+
+import opentelemetryTraceWaterfallFromLog from 'images/opentelemetry_screenshot-full_trace-waterfall-from-log.png'
+
+import opentelemetryLogCapturedinTraceSpan from 'images/opentelemetry_screenshot-full_log-captured-in-trace-span.png'
+
+
+
+
+
+Try out this Java tutorial to see what the New Relic platform can do with your OTLP data. You can quickly learn how to configure your OpenTelemetry SDK to optimize your observability experience in New Relic. You'll do all this using a demo Spring application that generates the nth number in the Fibonacci sequence.
```

Comment:
i might make it a little more clear about the demo app. At first read I thought the Fibonacci functionality had something to do with helping me instrument. Maybe something like: "For this tutorial, you'll be instrumenting a simple demo Spring app that generates Fibonacci numbers."

Reply:
@zuluecho9 I revised the first two paragraphs: 

Try out these Java tutorials to see what the New Relic platform can do with your OTLP data. We have three tutorials you can choose from, each one using the same demo Spring app that generates the nth number in the Fibonacci sequence. By working through these tutorials, you can learn skills to help you set up your own app with OpenTelemetry and New Relic.

While the three tutorials all use the same demo app, they have different approaches to help you become acquainted with OpenTelemetry and New Relic. Try out the tutorials that are interesting to you: